api/src/main/java/org/openmrs/module/bedmanagement/dao/impl/BedManagementDaoImpl.java

```diff
@@ -129,8 +129,9 @@ public List<BedPatientAssignment> getBedPatientAssignmentByVisit(String visitUui
 	public List<BedPatientAssignment> getCurrentAssignmentsByBed(Bed bed) {
```

Comment:
Can we have the `getCurrentAssignmentsByBed ` method called/not mocked, in a test?